### ISSUE TITLE
Support for new Identity

### ISSUE THREAD

**Issue by jakoss** [JudahGabriel/RavenDB.Identity#49]
.NET 8 will be released with new identity endpoints (i guess model will change under the hood as well). Do you have plans on supporting the new model? Details of changes: https://devblogs.microsoft.com/dotnet/improvements-auth-identity-aspnetcore-8/

**Comment by JudahGabriel** [comment]
Makes sense to support the new model. I'd certainly accept a PR to support the new model. Otherwise, I'll get it to it sometime after .NET 8 releases.
